[Bug]: master branch unit test fails in paramtable package

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version: d1de8ca
- Deployment mode(standalone or cluster):
- SDK version(e.g. pymilvus v2.0.0rc2):
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```


### Current Behavior

Unit test fails

### Expected Behavior

Unit test passes without error

### Steps To Reproduce

```markdown
Run unit test in paramtable package
```


### Milvus Log

```
=== RUN   TestGrpcClientParams
    grpc_param_test.go:59: Domain = datanode
    grpc_param_test.go:62: IP = 172.18.0.5
    grpc_param_test.go:65: Port = 21124
    grpc_param_test.go:67: Address = 172.18.0.5:21124
    grpc_param_test.go:70: ClientMaxRecvSize = 104857600
    grpc_param_test.go:82: ClientMaxSendSize = 104857600
    grpc_param_test.go:112: 
        	Error Trace:	grpc_param_test.go:112
        	Error:      	Not equal: 
        	            	expected: 20s
        	            	actual  : 3s
        	Test:       	TestGrpcClientParams
--- FAIL: TestGrpcClientParams (0.02s)
```

### Anything else?

_No response_

--- a/internal/util/paramtable/grpc_param.go
+++ b/internal/util/paramtable/grpc_param.go
@@ -41,7 +41,7 @@ const (
 	// Grpc Timeout related configs
 	DefaultDialTimeout      = 5000 * time.Millisecond
 	DefaultKeepAliveTime    = 10000 * time.Millisecond
-	DefaultKeepAliveTimeout = 3000 * time.Millisecond
+	DefaultKeepAliveTimeout = 20000 * time.Millisecond
 
 	ProxyInternalPort = 19529
 	ProxyExternalPort = 19530

